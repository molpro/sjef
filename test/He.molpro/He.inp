geometry={He}
basis,minao
rhf   
