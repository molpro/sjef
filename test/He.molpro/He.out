
 Primary working directories    : /var/folders/bg/1lcvbnvj7sv5vwxrqbqm40v00000gn/T//molpro.sVknxJweJA
 Secondary working directories  : /var/folders/bg/1lcvbnvj7sv5vwxrqbqm40v00000gn/T//molpro.sVknxJweJA
 Wavefunction directory         : /Users/peterk/wfu/
 Main file repository           : /var/folders/bg/1lcvbnvj7sv5vwxrqbqm40v00000gn/T//molpro.sVknxJweJA/

 id        : peterk

 Nodes                  nprocs
 Peters-Mac-mini-2.local    1

 Using customized tuning parameters: mindgm=1; mindgv=20; mindgc=4; mindgr=1; noblas=0; minvec=7
 default implementation of scratch files=df  


 Variables initialized (940), CPU time= 0.00 sec
 geometry={He}
 basis,minao
 rhf
 Commands  initialized (798), CPU time= 0.01 sec, 646 directives.
 Default parameters read. Elapsed time= 0.21 sec

 Checking input...
 Passed
1


                                         ***  PROGRAM SYSTEM MOLPRO  ***
                                       Copyright, TTI GmbH Stuttgart, 2015
                                    Version 2020.1 linked Apr 06 2020 11:55:04


 **********************************************************************************************************************************
 LABEL *                                                                                 
  64 bit serial version                                                                  DATE: 01-May-20          TIME: 17:51:54  
 **********************************************************************************************************************************

 SHA1:             a361670df9941fd611af5de7bb5329191040062f
 **********************************************************************************************************************************

 Memory per process:       400 MW
 Total memory per node:    400 MW

 GA preallocation disabled
 GA check disabled

 Variable memory set to 400.0 MW

 SETTING BASIS          =    MINAO


 Recomputing integrals since basis changed


 Using spherical harmonics

 Library entry HE     S MINAO                selected for orbital group  1


 PROGRAM * SEWARD (Integral evaluation for generally contracted gaussian basis sets)     Author: Roland Lindh, 1990

 Geometry written to block  1 of record 700

 Orientation using atomic masses  
 Molecule type: Atom
 Symmetry elements: X,Y,Z
 Rotational constants:       0.0000000      0.0000000      0.0000000 GHz  (calculated with average atomic masses)

 Point group  D2h 



 ATOMIC COORDINATES

 NR  ATOM    CHARGE       X              Y              Z

   1  HE      2.00    0.000000000    0.000000000    0.000000000

 NUCLEAR CHARGE:                    2
 NUMBER OF PRIMITIVE AOS:           6
 NUMBER OF SYMMETRY AOS:            6
 NUMBER OF CONTRACTIONS:            1   (    1Ag  +    0B3u +    0B2u +    0B1g +    0B1u +    0B2g +    0B3g +    0Au  )
 NUMBER OF INNER CORE ORBITALS:     0   (    0Ag  +    0B3u +    0B2u +    0B1g +    0B1u +    0B2g +    0B3g +    0Au  )
 NUMBER OF OUTER CORE ORBITALS:     0   (    0Ag  +    0B3u +    0B2u +    0B1g +    0B1u +    0B2g +    0B3g +    0Au  )
 NUMBER OF VALENCE ORBITALS:        1   (    1Ag  +    0B3u +    0B2u +    0B1g +    0B1u +    0B2g +    0B3g +    0Au  )


 NUCLEAR REPULSION ENERGY    0.00000000

 Eigenvalues of metric

         1 0.100E+01


 Contracted 2-electron integrals neglected if value below      1.0D-11
 AO integral compression algorithm  1   Integral accuracy      1.0D-11

     0.262 MB (compressed) written to integral file (100.0%)


 NUMBER OF SORTED TWO-ELECTRON INTEGRALS:          1.     BUFFER LENGTH:  32768
 NUMBER OF SEGMENTS:   1  SEGMENT LENGTH:          1      RECORD LENGTH: 524288

 Memory used in sort:       0.56 MW

 SORT1 READ           1. AND WROTE           1. INTEGRALS IN      1 RECORDS. CPU TIME:     0.00 SEC, REAL TIME:     0.00 SEC
 SORT2 READ           1. AND WROTE           1. INTEGRALS IN      1 RECORDS. CPU TIME:     0.00 SEC, REAL TIME:     0.00 SEC

 FILE SIZES:   FILE 1:    30.2 MBYTE,  FILE 4:     4.2 MBYTE,   TOTAL:     34.4 MBYTE

 OPERATOR DM      FOR CENTER  0  COORDINATES:    0.000000    0.000000    0.000000


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       29.92       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

 PROGRAMS   *        TOTAL       INT
 CPU TIMES  *         0.33      0.26
 REAL TIME  *         1.20 SEC
 DISK USED  *        34.66 MB
 **********************************************************************************************************************************


 PROGRAM * RHF-SCF (CLOSED SHELL)       Authors: W. Meyer, H.-J. Werner


 NUMBER OF ELECTRONS:       1+    1-    SPACE SYMMETRY=1    SPIN SYMMETRY: Singlet 
 CONVERGENCE THRESHOLDS:    1.00E-05 (Density)    1.00E-07 (Energy)
 MAX. NUMBER OF ITERATIONS:       60
 INTERPOLATION TYPE:            DIIS
 INTERPOLATION STEPS:              2 (START)      1 (STEP)
 LEVEL SHIFTS:                  0.00 (CLOSED)  0.00 (OPEN) 



 Orbital guess generated from atomic densities. Full valence occupancy:    1

 Molecular orbital dump at record        2100.2

 Initial occupancy:   1

 ITERATION    DDIFF          GRAD             ENERGY        2-EL.EN.            DIPOLE MOMENTS         DIIS   ORB.
    1      0.000D+00      0.000D+00        -2.86115334      2.051807    0.00000    0.00000    0.00000    0    start
    2      0.000D+00      0.000D+00        -2.86115334      2.051807    0.00000    0.00000    0.00000    0    orth

 Final occupancy:   1

 !RHF STATE  1.1 Energy                -2.861153344782
 Nuclear energy                         0.00000000
 One-electron energy                   -3.88705707
 Two-electron energy                    1.02590373
 Virial quotient                       -0.99999991
 !RHF STATE  1.1 Dipole moment          0.00000000     0.00000000     0.00000000
 Dipole moment /Debye                   0.00000000     0.00000000     0.00000000

 Orbital energies:

         1.1
     -0.917625


 HOMO      1.1    -0.917625 =     -24.9698eV


 **********************************************************************************************************************************
 DATASETS  * FILE   NREC   LENGTH (MB)   RECORD NAMES
              1      19       29.92       500      610      700      900      950      970     1000      129      960     1100   
                                          VAR    BASINP    GEOM    SYMINP    ZMAT    AOBASIS   BASIS     P2S    ABASIS      S 
                                         1400     1410     1200     1210     1080     1600     1650     1300     1700   
                                           T        V       H0       H01     AOSYM     SMH    MOLCAS    ERIS     OPER   

              2       4        2.73       700     1000      520     2100   
                                         GEOM     BASIS   MCVARS     RHF  

 PROGRAMS   *        TOTAL   RHF-SCF       INT
 CPU TIMES  *         0.38      0.05      0.26
 REAL TIME  *         1.28 SEC
 DISK USED  *        34.66 MB
 **********************************************************************************************************************************
 Molecular orbitals read from record     2100.2  Type=RHF/CANONICAL (state 1.1)
 Occupation numbers read from record     2100.2  Type=RHF/RHF (state 1.1)
 Orbital energies read from record       2100.2  Type=RHF/CANONICAL (state 1.1)
 Redundancy group numbers read from rec  2100.2  Type=RHF/RHF (state 1.1)

 RHF/MINAO energy=     -2.861153344782

         RHF-SCF
     -2.86115334
 **********************************************************************************************************************************
 Molpro calculation terminated
